Angular/src/app/login/login.component.css

```diff
@@ -0,0 +1,414 @@
+/*//////////////////////////////////////////////////////////////////
```

Comment:
Where did you get all this CSS from? Is it all really needed?

Reply:
removed all the unneeded classes. From a open source style example